openllm-core/src/openllm_core/config/configuration_mpt.py

```diff
@@ -3,7 +3,7 @@
 
 import openllm_core
 
-from openllm_core._prompt import process_prompt
+from openllm_core.prompts._prompt import process_prompt
```

Comment:
/s/openllm_core.prompts._prompt/openllm_core.prompts

also it should just be `openllm_core.prompt`, no need to be plural 😄